ansible/group_vars/all

```diff
@@ -498,7 +498,7 @@ durationChecker:
     spi: "{{ duration_checker_spi | default('') }}"
     timeWindow: "{{ duration_checker_time_window | default('1 d') }}"
 
-enable_scheduler: "{{ scheduler_enable | default(false) }}"
+enable_scheduler: "{{ scheduler_enable | default(true) }}"
```

Comment:
The scheduler test failure of this PR was because the scheduler was not enabled.
As we would enable the scheduler by default with https://github.com/apache/openwhisk/pull/5463, I rebased this PR from it.